src/main/java/firebase/FirebaseInitializer.java

```diff
@@ -6,12 +6,16 @@
 import com.google.auth.oauth2.GoogleCredentials;
 import com.google.firebase.cloud.FirestoreClient;
 
+import java.io.FileInputStream;
+
 public class FirebaseInitializer {
 
     public static Firestore initializeFirebase() {
         try {
+            FileInputStream serviceAccount = new FileInputStream(" add your path to the service account");
```

Comment:
This will be different for everyone, is there a way to maybe make it an environment variable?